google/cloud/pubsub/google_cloud_cpp_pubsub.bzl

```diff
@@ -111,6 +111,7 @@ google_cloud_cpp_pubsub_hdrs = [
     "internal/subscription_message_source.h",
     "internal/subscription_session.h",
     "internal/tracing_batch_sink.h",
```

Comment:
Why not add this function to `google/cloud/internal/opentelemetry.h`? Is that file too much of a dumpster fire?

Reply:
Yes and currently only needed for pubsub. If we want it for others, we can move it to google/cloud/internal/X

Make something other than opentelemetery. That file is pretty big.